wiki/osu!wiki_Contribution_Guide/id.md

```diff
@@ -0,0 +1,106 @@
+# Panduan Kontribusi osu!wiki
+
+Terima kasih atas minat anda untuk membuat osu!wiki menjadi lebih baik! Saat ini, berkontribusi pada osu!wiki akan membutuhkan lebih banyak waktu dan usaha untuk membuat perubahan yang anda lakukan benar-benar diterapkan. Panduan ringkas ini ditujukan untuk pendatang baru yang belum pernah menggunakan GitHub dan belum mengenal alur kerja kontribusi osu!wiki sebelumnya. Jika anda sudah pernah berpengalaman menggunakan GitHub, tidak disalahkan untuk melewatkan panduan ini dan mengikuti alur kerja fitur branch untuk pembaruan konten di repositori osu!wiki.
+
+Jika, selama bagian apa pun, anda bingung dan/atau memerlukan bantuan lebih lanjut, jangan ragu mengirim pesan ke [Discord osu!dev](https://discord.gg/ppy) di kanal `#osu-wiki`.
+
+## Memulai
+
+### Mendaftar
+
+1\. [Mendaftar](https://github.com/join) akun GitHub jika anda belum memilikinya.
+
+2\. [Masuk](https://github.com/login).
+
+3\. Lanjutkan ke [Forking](#forking).
+
+### Forking
+
+1\. Kunjungi [repo `osu-wiki`](https://github.com/ppy/osu-wiki).
+
+2\. Klik `Fork`.
+
+![](img/fork.jpg)
+
+3\. Klik `Fork` sekali lagi untuk diarahkan ke fork anda.
+
+4\. Jika anda melihat sesuatu yang mirip dengan gambar di bawah ini berarti anda telah membuat fork dari repo `ppy/osu-wiki` dan sekarang anda berada di fork anda.
+
+![](img/forked.jpg)
+
+5\. Lanjutkan ke [Syncing your fork](#syncing-your-fork).
+
+### Menyinkronkan fork anda
+
+1\. Kunjungi fork repo `osu-wiki` anda.
+
+2\. Di atas file, ada beberapa teks yang memberi tahu anda apakah branch dalam keadaan "behind", "ahead", atau "even dengan `ppy:master.` Jika branch dalam keadaan "behind" atau "ahead and behind" dengan jumlah behind-commits di belakangnya, branch anda belum diperbaharui. 
```

Comment:
after every english word in quotation marks, add their translation in brackets

also, you forgot a quotation mark after '"even dengan...'